src/cmd/getM365/getItem.go

```diff
@@ -148,6 +148,7 @@ func runDisplayM365JSON(
 
 		fmt.Println(string(array))
 
+		//lint:ignore SA4004 only expecting one item
 		return nil
```

Comment:
Why is the lint ignore necessary here?

Reply:
We will unconditionally exit from the loop here after the first iter and that is flagged as a warning by default under [SA4004](https://staticcheck.io/docs/checks/#SA4004) check.